tracer/src/Datadog.Trace/Generated/net6.0/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.AspectsDefinitions.AspectsDefinitionsGenerator/AspectsDefinitions.g.cs

```diff
@@ -11,17 +11,17 @@ namespace Datadog.Trace.ClrProfiler
     internal static partial class AspectDefinitions
```

Comment:
these three auto generated `AspectsDefinitions` are the only ASM-related changes. They were modified automatically when I built the tracer, and seem to just be a re-ordering of pre-existing elements.

Let me know if this shouldn't be checked in.